lib/codelog/command/bump.rb

```diff
@@ -6,16 +6,20 @@ class Bump
       VALID_VERSION_TYPES = ['major', 'minor', 'patch'].freeze
       INITIAL_RELEASE_VERSION = '0.0.0'.freeze
 
-      def self.run(version_type, release_date)
-        Codelog::Command::Bump.new.run version_type, release_date
+      def self.run(version_type, release_date, options)
+        Codelog::Command::Bump.new.run version_type, release_date, options
       end
 
-      def run(version_type, release_date)
+      def run(version_type, release_date, options)
```

Comment:
Unfortunatelly  i had to duplicate logic and pass the options here, it was because the bump command needs to run its own  logic before calling the next commands.